scripts/dev/minio-init.sh

```
@@ -3,13 +3,19 @@ set -euo pipefail
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 REPO_ROOT="$(cd "${SCRIPT_DIR}/../.." && pwd)"
-PROJECT_NAME="${PROJECT_NAME:-new-engine}"
+ROOT_DIR="$REPO_ROOT"
```

Shellcheck (0.11.0)

[warning] ROOT_DIR appears unused. Verify use (or export if used externally).

(SC2034)